### ISSUE TITLE
Crash when clicking on Repair

### ISSUE THREAD

**Issue by A-T-L-A-S** [mickelus/tetra#104]
## Bug Report

**Observed Behaviour**
When clicking on repair (in my case with a diamond pickaxe) in the workbench my game immediatly crashes.
Tested it with a clean install of just forge and Tetra with the same result.

Crash log: https://pastebin.com/adgVeK4F

**Expected Behaviour**
No crash

**Minimal setup needed to reproduce**
- Forge version: 1.14.4forge-28.1.111
- Tetra version: tetra-1.14.4-1.11.1
- Tetra configuration: ---clean install---
- Other mods: none

**Steps to reproduce**
-place a workbench
-open it and place any tool inside
-click on repair

**Comment by TempAccTrowaway** [comment]
Wow, what a coincidence. I downloaded this excellent mod a little while ago, and tested it in a new world with creative mode. So I got the same error on the basic log mallet, just clicked repair, after exploring what the menu offers, and then instant crash. I do not know what could be causing this issue, I am running a couple of mods too, so maybe one of them. But it seems like Atlas got the same error even with a clean install and no mods. I can provide a crash log too: https://pastebin.com/rick3FVa
